packages/aws-cdk/lib/api/hotswap/ecs-services.ts

```diff
@@ -167,13 +189,47 @@ export async function isHotswappableEcsServiceChange(
   return ret;
 }
 
+function mergeTaskDefinitions(patch: {containerDefinitions: {[key:string]: any}[]}, target: AWS.ECS.DescribeTaskDefinitionResponse) {
+  const src = patch.containerDefinitions;
+  const dst = target.taskDefinition?.containerDefinitions;
+  if (dst === undefined) return;
```

Comment:
```suggestion
  if (dst === undefined) {
    return;
  }
```

Reply:
OK! Shouldn't we enforce this in eslintrc?

https://github.com/aws/aws-cdk/blob/1807f5754885e4b1b1c8d12ca7a1cc7efab9ef2c/tools/%40aws-cdk/cdk-build-tools/config/eslintrc.js#L65